modules/oe_theme_helper/oe_theme_helper.post_update.php

```diff
@@ -257,3 +258,23 @@ function oe_theme_helper_post_update_20014() {
     $block->save();
   }
 }
+
+/**
+ * Create oe_theme_main_content view mode for Iframe media.
+ */
+function oe_theme_helper_post_update_20015() {
+  if (!\Drupal::moduleHandler()->moduleExists('oe_media_iframe')) {
+    // Since core.entity_view_display.media.iframe.oe_theme_main_content is
+    // optional config we have to ensure that module is enabled.
+    return t('Module oe_media_iframe is not enabled.');
+  }
+
+  $file_storage = new FileStorage(drupal_get_path('theme', 'oe_theme') . '/config/optional');
```

Comment:
There isn't `oe_theme/config/post_updates` folder so I implemented it as in oe_theme_helper_post_update_20006().

Reply:
That update path is old, please introduce the folder with the proper procedure we use elsewhere.

Reply:
Folder config/post_updates/20015_create_view_display_media_iframe has been created.